petastorm/spark/spark_dataset_converter.py

```diff
@@ -119,19 +120,55 @@ def make_tf_dataset(self,
             preproc_parallelism=preproc_parallelism
         )
 
-    def make_torch_dataloader(self):
+    def make_torch_dataloader(self,
+                              batch_size=32,
+                              num_epochs=None,
+                              workers_count=None,
+                              cur_shard=None,
+                              shard_count=None,
+                              preprocess_pandas_fn=None,
+                              **petastorm_reader_kwargs):
         """
         Make a PyTorch DataLoader.
 
         This method will do the following two steps:
           1) Open a petastorm reader on the materialized dataset dir.
           2) Create a PyTorch DataLoader based on the reader created in (1)
 
+        :param batch_size: The number of items to return per batch
+        :param num_epochs: An epoch is a single pass over all rows in the
+            dataset. Setting ``num_epochs`` to ``None`` will result in an
+            infinite number of epochs.
+        :param workers_count: An int for the number of workers to use in the
+            reader pool. This only is used for the thread or process pool.
+            Defaults to None, which means using the default value from
+            `petastorm.make_batch_reader()`.
+        :param cur_shard: An int denoting the current shard number. Each node
+            reading a shard should pass in a unique shard number in the range
+            [0, shard_count). shard_count must be supplied as well. Defaults to
+            None
+        :param shard_count: An int denoting the number of shards to break this
+            dataset into. Defaults to None
+        :param preprocess_pandas_fn: Process one row group as a pandas
+            DataFrame/Series (single column) and return a pandas
+            DataFrame/Series
+        :param petastorm_reader_kwargs: all the arguments for
+            `petastorm.make_batch_reader()`.
+
         :return: a context manager for a `torch.utils.data.DataLoader` object.
                  when exit the returned context manager, the reader
                  will be closed.
         """
-        return _torch_dataset_context_manager(self.cache_dir_url)
+        return _torch_dataset_context_manager(
```

Comment:
Use TorchDatasetContextManager class name

Reply:
I change it in PR #25